meetings/read_minutes.py

```diff
@@ -71,10 +71,16 @@ def format_topics(summary):
 
 def format_actions(action_items):
     actions = []
+    last_action = None
     for line in action_items.split("\n"):
         if line.startswith("* "):
             assignee, action = line[2:].split(" ", 1)
+            last_action = len(actions)
             actions.append(f"- [ ] @{assignee} {action}")
+        elif line.startswith("  ") and last_action is not None:
+            actions[last_action] += f" {line[2:]}"
```

Comment:
```suggestion
            actions[last_action - 1] += f" {line[2:]}"
```
I mean if we have `actions = [0, 1, 2]` it has lenght of 3, so won't we get something like below?
```
>>> a = ['a', 'b', 'c']
>>> a[len(a)] += 'd'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range
```

Reply:
@Andersson007 `last_action` is the length taken before the append, so it doesn't need the adjustment you suggest

Reply:
@willthames ah, ok, thanks for clarifying

Reply:
feel a bit sleepy:)